### PR TITLE
feat(os): Prod Development doc - Added Design Scoping meeting info + some md styling updates

### PR DESCRIPTION
**Pull request by angelaliu22**
**Why**
Reviewed the Product Development process doc for any additions from the design side. The only major addition relevant now was adding a description of the ad-hoc meeting for Design Scoping & Review. 

**What**
- Added Design Scoping & Review meeting purpose and outcome in the Ad-Hoc meeting section
- Updated a bunch of md styling and some grammar throughout the doc

### CODE REVIEW

File: docs/process/development.md [hunk 1]
```diff
@@ -207,39 +222,50 @@ Typical agenda items are:
...
+
 **Purpose**: Discussion of regulatory and legislative requirements at the national and regional level with Tusk, one of our investors.
 
 **Output**:  Ongoing notes. Action items on specific regulatory changes as needed.
 
 ### Ad-hoc
+#### Design review meeting
+* **Duration**: As long as it takes (typically starting with 90 minutes)
+
+**Purpose**: To denmark the end of large scope design work for a project (small tweaks, edge cases, etc can still be flushed out after meeting), and review user flows and questions with wider product team. It is expected that intermittent discussions and reviews with the product and technical teams take place prior to this meeting via Figma comments or Slack threads. However, there is still a time when development needs to start, and a formal review meeting with the relevant parties is called. To make the most of this meeting, designer should resolve as many questions as possible via Figma comments, slack discussions, and individual research ahead of time, and reserve synchronous time to meatier discussions.
```
Comment (amjarruda) [inline docs/process/development.md:247]:
```suggestion
**Purpose**: To demark the end of large scope design work for a project (small tweaks, edge cases, etc can still be flushed out after meeting), and review user flows and questions with wider product team. It is expected that intermittent discussions and reviews with the product and technical teams take place prior to this meeting via Figma comments or Slack threads. However, there is still a time when development needs to start, and a formal review meeting with the relevant parties is called. To make the most of this meeting, designer should resolve as many questions as possible via Figma comments, slack discussions, and individual research ahead of time, and reserve synchronous time to meatier discussions.
```

Comment (amjarruda) [reply docs/process/development.md:247]: Think this is a small typo.

**Review by amjarruda — APPROVED** [review]
Looks great, good additions.

**Comment by angelaliu22** [comment]
@amjarruda thanks for the review! I resolved comments and merge conflicts. I don't have write access so please go ahead and merge whenever you have a chance!

**Comment by amjarruda** [comment]
All set!

**Pull request merged**